astropy/utils/iers/iers.py

```diff
@@ -305,6 +312,82 @@ def pm_xy(self, jd1, jd2=0., return_status=False):
         return self._interpolate(jd1, jd2, ['PM_x', 'PM_y'],
                                  self.pm_source if return_status else None)
 
+    def _leap_seconds(self):
+        """Get a table with year, month, day on which leap seconds occur.
+
+        Find leap seconds from their jumps in UT1-UTC, and create a new
+        table with corresponding year, month, and day, as well as the
+        cumulative offset.  The latter should equal TAI-UTC plus some
+        unknown zero point.
+        """
+        delta = np.diff(self['UT1_UTC'])
+        sel = np.abs(delta) > 0.5 * u.s
+        # Recalculate year, month, day as IERS_A only stores a 2-digit year.
+        t = Time(self['MJD'][1:][sel], format='mjd').ymdhms
+        return QTable([t['year'], t['month'], t['day'],
+                       np.cumsum(np.round(delta[sel].value))],
+                      names=['year', 'month', 'day', 'offset'])
+
+    def update_erfa_leap_seconds(self):
+        """Add any leap seconds not already present to the ERFA table.
+
+        This method looks for the presence of leap seconds in the IERS
+        table from jumps in UT1-UTC.  It then matches those with the
+        leap seconds present in the ERFA table, and extends the latter
+        as necessary.
+
+        Returns
+        -------
+        n_update : int
+            The number of leap seconds added to the ERFA table.
+
+        Raises
+        ------
+        ValueError
+            If the leap seconds found are not on 1st of January or July,
+            if no matching leap seconds were found, or if the matches are
+            inconsistent.  This would normally suggested a currupted
+            ERFA or IERS table.  The ERFA table can be reset using
+            `~astropy._erfa.set_leap_seconds()`.
```

Comment:
```suggestion
            ``astropy._erfa.set_leap_seconds()``.
```

as _erfa is not part of the public API I think.

Reply:
That is a good point. Given that, maybe it is better to include an explicit way to reset the table inside `iers`.

Reply:
That is a good point. Given that, maybe it is better to include an explicit way to reset the table inside `iers`.

Reply:
That is a good point. Given that, maybe it is better to include an explicit way to reset the table inside `iers`.

Reply:
I now changed the method so that it takes an argument that allows one to initialize the table (and also so that it returns the final one).